.gitignore

```diff
@@ -11,6 +11,7 @@ lib
 out
 sqldelight-gradle-plugin/src/test/*/gradle.properties
 sqldelight-gradle-plugin/src/test/integration-android-library/src/
+kotlin-js-store
```

Comment:
FWIW, I was under the impression that this folder should be commited in source control for reproducible builds. See https://kotlinlang.slack.com/archives/C0B8L3U69/p1639508052221600 and https://youtrack.jetbrains.com/issue/KT-34014